examples/junit_xml_enricher/enrich_junit_xml.py

```
@@ -0,0 +1,258 @@
+import sys
+from pathlib import Path
+from typing import Any
+from xml.etree import ElementTree as ET
+
+from dotenv import load_dotenv
+
+# Add sibling example directory to path for shared utilities
+_CONFTEST_DIR = Path(__file__).resolve().parent.parent / "pytest_junitxml"
+sys.path.insert(0, str(_CONFTEST_DIR))
+
+from conftest_junit_ai_utils import (  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)